lib-ce/emqx_dashboard/src/emqx_dashboard_admin.erl

```diff
@@ -146,17 +146,26 @@ update_pwd(Username, Fun) ->
     Trans = fun() ->
                     User =
                     case lookup_user(Username) of
-                    [Admin] -> Admin;
-                    [] ->
-                           mnesia:abort(<<"Username Not Found">>)
+                        [Admin] -> Admin;
+                        [] ->
+                            mnesia:abort(<<"Username Not Found">>)
                     end,
                     mnesia:write(Fun(User))
             end,
     return(mnesia:transaction(Trans)).
 
 
 -spec(lookup_user(binary()) -> [mqtt_admin()]).
-lookup_user(Username) when is_binary(Username) -> mnesia:dirty_read(mqtt_admin, Username).
+lookup_user(Username) when is_binary(Username) ->
+    IsDefaultUser = binenv(default_user_username) =:= Username,
+    case mnesia:dirty_read(mqtt_admin, Username) of
```

Comment:
Nit: double reading of the database can be eliminated.

Reply:
this is to avoid having to do transaction most of the times during boot up.

Reply:
ah, duplicated dirty_read as line 168. true. will fix.